code-challenges.test.js

```diff
@@ -14,38 +16,125 @@
 
 // a) Create a test with an expect statement using the variable provided.
 
-const people = [
-  { name: "ford prefect", occupation: "a hitchhiker" },
-  { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
-  { name: "arthur dent", occupation: "a radio employee" }
-]
-// Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]
-
+// 1a) Pseudo Code:
+// Copy and paste test code template into assignment (make sure nothing is inside initial template)
+// Read problem statement and understand inputs/outputs (inputs need to be directly inserted into test code)
+// Create relevant info for describe, it, expect, and toEqual methods
+// Test code output to ensure it is failing red (should get a reference error saying function is not defined)
+
+describe("capitalizerPartTwo", () => {
+  it("a function which takes in an array of objects, which returns an array of strings which capitalizes the name of each person and combines it with their occupation", () => {
+    expect(
+      capitalizerPartTwo([
+        { name: "ford prefect", occupation: "a hitchhiker" },
+        { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
+        { name: "arthur dent", occupation: "a radio employee" },
+      ])
+    ).toEqual([
+      "Ford Prefect is a hitchhiker.",
+      "Zaphod Beeblebrox is president of the galaxy.",
+      "Arthur Dent is a radio employee.",
+    ]);
+  });
+});
```

Comment:
Your test looks beautiful. Your approach is valid but you can also just use the test variable as is.